build.gradle

```diff
@@ -78,6 +78,8 @@ dependencies {
   testCompile 'org.assertj:assertj-core:3.11.1'
   testCompile 'org.mockito:mockito-core:3.3.3'
   testCompile 'org.testcontainers:postgresql:1.13.0'
+
+  compile 'com.smoketurner.dropwizard:graphql-core:2.0.7-1'
```

Comment:
_Minor_: I would keep `compile`-level and `testCompile`-level grouped together in the build file.